android/modules/calendar/src/java/ti/modules/titanium/calendar/EventProxy.java

```diff
@@ -117,7 +120,7 @@ public static String getExtendedPropertiesUri()
 			event.hasAlarm = !eventCursor.getString(7).equals("0");
 			event.status = eventCursor.getInt(8);
 			event.visibility = eventCursor.getInt(9);
-
+			event.setRecurrenceRules(eventCursor.getString(10), eventCursor.getInt(11));
```

Comment:
could this break when pulling up existing events with no recurrence?

Reply:
Apparently it could. I did not get any exception, but it is documented as "implementation-defined"
https://developer.android.com/reference/android/database/Cursor.html#getString(int)
I will add an exception handling.